examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,85 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito compiler\n",
+    "\n",
+    "The following series of notebooks will describe these compilation passes in a hands-on fashion. We will explore the heart of the `Operator` class, which orchestrates code generation and compilation. For more details, see the [Operator class](https://github.com/opesci/devito/blob/master/devito/operator.py).\n",
```

Comment:
"these compilation passes" -- which ones? the sentence is a bit odd...

Reply:
oh yes. i shouldn't have moved those paragraphs orders in the first place.